### PR TITLE
Adding a trip model and cleaning up model constants

### PR DESCRIPTION
**Pull request by kravisankaran**
Adding a new trip model with some clean up activities

### CODE REVIEW

**Review by mike-zaharchenko17 — COMMENTED** [review]
Added comments

File: trip/admin.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```
[the remark this answers is not in the document]
Comment (kravisankaran) [reply trip/admin.py:3]: Good catch

File: trip/models.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+
+# Create your models here.
+class Trip(models.Model):
+    destination_city = models.CharField(max_length=500)
+    destination_country = models.CharField(max_length=500)
+    start_trip = models.DateTimeField("start_trip")
+    end_trip = models.DateTimeField("end_trip")
+    travel_type = ChoiceArrayField(
+        models.CharField(max_length=30, choices=TRAVEL_TYPE), default=list
```
[the remark this answers is not in the document]
Comment (kravisankaran) [reply trip/models.py:14]: When we discussed this on friday - we were talking about having a trigger here to control if they want to match with others for this trip

[the remark this answers is not in the document]
Comment (mike-zaharchenko17) [reply trip/models.py:14]: Cool- I think we should make what the choices do a little clearer on the front end when we get around to making it though

**Review by mike-zaharchenko17 — APPROVED** [review]
LGTM

**Pull request merged**